molecule/shell.py

```diff
@@ -87,7 +88,7 @@ def _allowed(ctx, param, value):  # pragma: no cover
 @click.group()
 @click.option(
     '--debug/--no-debug',
-    default=False,
+    default=MOLECULE_DEBUG,
```

Comment:
@ssbarnea @decentral1se I just realized that it's Click. And it supports env vars out of the box. There's `envvar='MOLECULE_DEBUG'` argument to the decorator and also there's a global prefix you can set up for options/args.
https://click.palletsprojects.com/en/7.x/options/#values-from-environment-variables